packages/core/test/unit/dialects/mssql/query-generator.test.js

```diff
@@ -180,7 +184,7 @@ if (current.dialect.name === 'mssql') {
       expectsql(this.queryGenerator.addColumnQuery('myTable', 'myColumn', { type: 'VARCHAR(255)', comment: 'This is a comment' }), {
         mssql: 'ALTER TABLE [myTable] ADD [myColumn] VARCHAR(255) NULL; EXEC sp_addextendedproperty '
```

Comment:
Nitpicky, but we could use backticks (`) here to reduce the amount of escaping needed